features/networking/pod.feature

```diff
@@ -190,3 +190,126 @@ Feature: Pod related networking scenarios
     Then the step should succeed
     And the output should not contain "ingress_policing_rate: 1953"
 
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23890
+  @admin
+  Scenario: A pod with or without hostnetwork cannot access the MCS port 22623 or 22624 on the master
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    Given I have a project
+    #pod-for-ping will be a non-hostnetwork pod
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    And I have a pod-for-ping in the project
+    And the pod named "hello-pod" becomes ready
```

Comment:
remove this step. Step `I have a pod-for-ping in the project` already waits for the pod. Also no need to expose implementation detail like pod name `hello-pod` in scenario. There is already clipboard populated. `cb.ping_pod = pod("hello-pod")`, so when you need pod name, you can use `cb.ping_pod.name`.

Reply:
Yep i agree, Alex. Was taken care of. Thanks!